F1WM.IntegrationTests/TracksTests.cs

```diff
@@ -15,12 +17,39 @@ public async Task ShouldGetTrackRecords(TrackRecordsTestData data)
 				data.Expected);
 		}
 
+		[Theory]
+		[JsonData("tracks", "track-records.json")]
```

Comment:
This doesn't make any sense. You provide data from track-records.json while expecting TracksSummaryTestData. You should create a separate file with track summary test data.